models/concerns/as_json.rb

```diff
@@ -0,0 +1,239 @@
+# Copyright © Mapotempo, 2021
+#
+# This file is part of Mapotempo.
+#
+# Mapotempo is free software. You can redistribute it and/or
+# modify since you respect the terms of the GNU Affero General
+# Public License as published by the Free Software Foundation,
+# either version 3 of the License, or (at your option) any later version.
+#
+# Mapotempo is distributed in the hope that it will be useful, but WITHOUT
+# ANY WARRANTY; without even the implied warranty of MERCHANTABILITY
+# or FITNESS FOR A PARTICULAR PURPOSE.  See the Licenses for more details.
+#
+# You should have received a copy of the GNU Affero General Public License
+# along with Mapotempo. If not, see:
+# <http://www.gnu.org/licenses/agpl.html>
+#
+module ContainedPointAsJson
+  extend ActiveSupport::Concern
+
+  def as_json(options = {})
+    point = super
+
```

Comment:
AcriveModel initilises options with `nil`
and we need to call `super` with options so that the lower level as_json gets the options  
https://apidock.com/rails/v3.2.1/ActiveModel/Serializers/JSON/as_json